go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/canonical/k8s-dqlite
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.10
```

Comment:
why is this needed? Are we really relying on this specific patch version to build this?

Reply:
I believe `go mod tidy` started adding this, see https://github.com/golang/go/issues/65847